lib/suse/connect/client.rb

```diff
@@ -46,11 +46,8 @@ def register_product(product, install_release_package = true)
         service = activate_product(product, @config.email)
 
         System.add_service(service)
+        Zypper.install_release_package(product.identifier) if install_release_package
```

Comment:
AFAIK this was the only place where we called `Zypper.refresh_services`. Maybe you want to remove the method in `zypper.rb` too?

Reply:
But I'm not sure if its also used by zypper or anybody else. That's why I left it in there (since I didn't created it)

Reply:
In case your sure it's safe to be removed, I remove it! :+1:

Reply:
Hm the method seems to be three years old, but now I am tbh also not sure if something external might call it. Before removing it maybe explain why we don't want this call here anymore, it's not entirely clear to me.